Geometry/HGCalCommonData/plugins/DDHGCalCell.cc

```diff
@@ -67,7 +67,7 @@ void DDHGCalCell::initialize(const DDNumericArguments& nArgs,
                                  << " Truncated " << truncCN_.size() << ":" << truncSensN_.size() << " Extended "
                                  << extenCN_.size() << ":" << extenSensN_.size() << " Corners " << cornrCN_.size()
                                  << ":" << cornrSensN_.size();
-  if ((truncCN_.size() != 3) || (extenCN_.size() != 3) || (cornrCN_.size() != 6))
+  if ((truncCN_.size() != 6) || (extenCN_.size() != 6) || (cornrCN_.size() != 12))
```

Comment:
Should these magic numbers be constants?

Reply:
I second the comment of @cvuosalo : these sizes are only defined as such for Geometry/​HGCalCommonData/​data/​hgcalcell/​vtest/​hgcalcell.xml , and you are hardcoding in a plugin an adaptation for a `vtest` geometry (and by the way, the LogWarning below is not even updated with the new sizes).
I think that either the plugin is able to retrieve those sizes from the geometry, or they must be made configurable and adapted as the specific data you are testing.

Reply:
This is not a good warning - I am removing this.